import/popups/AlertDialog.qml

```diff
@@ -19,6 +19,8 @@ UserPrompt {
     //% "Ok"
     acceptText: qsTrId("sailfish_components_webview_popups-he-accept_alert")
 
+    backNavigation: false
```

Comment:
Nope... If you like to "fix" this like this... I'd suggest just rename "Cancel" to "Ok". This does not affect to the reply messages that we're sending back to the engine.

Reply:
Could you elaborate on your reasons please? Do you mean have two buttons with "OK" on them?

Reply:
Sure... It's just dismissing it from both edges. As we're using dialogs (pages) for this would be good not to break navigation patterns (having two OK "buttons" is not bad).

That said, using dialogs for this is just plain wrong as it should be bound to the context where is was triggered not to some platform navigation logic. Further, a fullscreen dialog pulls user's focus away from the context where the alert was triggered this way causing unnecessary context switches (confusion).

Reply:
Any updates?

Reply:
I've updated the PR to allow left-swiping again. Martin agreed this should still be supported. He suggested to just remove the text from the "Cancel" button, which this PR now reflects.